test/helpers/client.js

```diff
@@ -44,17 +44,22 @@ class Client {
         this._respondWithCallback(JSON.parse(data));
     }
 
+    _setupMessageEvents() {
+        this.ws.on('open', this._onOpen.bind(this));
+        this.ws.on('close', this._onClose.bind(this));
+        this.ws.on('message', this._onMessage.bind(this));
+    }
+
     _checkError(error) {
         // Assumes that an instance of the server is not running, so lets start one.
-        if (error.errno === "ECONNREFUSED") {
+        if (error.errno === 'ECONNREFUSED') {
             if (!this.child) {
-                this.child = fork('main.js', ['grunt', '--debug', 'silly'])
+                this.child = fork('main.js', ['grunt', '--debug', 'silly']);
```

Comment:
My point is that it would be trivial to update any of the current default modules and make them break because of this line. A fork start of Concierge is assumed to be a `require('concierge-bot')` which has very very different require behaviour to starting it from the CLI (which all of our tests assume). For something like KPM this could result in outright failure of all tests on the wrong version of node.

So why do we even need to fork into it? Your hang up issues are probably due to your increadibly short timeout of 5s (which I commented on earlier)